### PR TITLE
Update metering in KKP 2.21

### PR DESCRIPTION
**Pull request by WeirdMachine**
Backports metering docs to KKP 2.21 

Add new added Average Cluster Machines value to docs

### CODE REVIEW

**Comment by WeirdMachine** [comment]
/hold

Wait for KKP 2.21 patch release

**Review by embik — CHANGES_REQUESTED** [review]
I think there was a copy-paste issue, other than that it looks good.

File: content/kubermatic/v2.21/tutorials-howtos/metering/_index.en.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ For that to work properly the s3 endpoint needs to be available from the browser
 Using the dashboard, configuring the Metering tool becomes a breeze.
 Open the [Admin Panel][admin-panel] and choose the **Metering** tab on the left side.
 
-![Navigation to Metering configuration and reports](/img/kubermatic/v2.21/tutorials/metering_admin_panel_location.png?classes=shadow,border "Navigation to Metering configuration and reports")
+![Navigation to Metering configuration and reports](/img/kubermatic/main/tutorials/metering_admin_panel_location.png?classes=shadow,border "Navigation to Metering configuration and reports")
```
[the remark this answers is not in the document]
Comment (WeirdMachine) [reply content/kubermatic/v2.21/tutorials-howtos/metering/_index.en.md:42]: Ai copy paste error, thanks for catching this.

**Review by embik — APPROVED** [review]

**Pull request merged**